zcash_client_backend/src/data_api.rs

```diff
@@ -794,6 +794,69 @@ impl<NoteRef> SpendableNotes<NoteRef> {
     }
 }
 
+/// Metadata about the structure of the wallet for a particular account.
+///
+/// At present this just contains counts of unspent outputs in each pool, but it may be extended in
+/// the future to contain note values or other more detailed information about wallet structure.
+///
+/// Values of this type are intended to be used in selection of change output values. A value of
+/// this type may represent filtered data, and may therefore not count all of the unspent notes in
+/// the wallet.
+pub struct WalletMeta {
+    sapling_note_count: usize,
+    #[cfg(feature = "orchard")]
```

Comment:
Do we really need to gate this on the `orchard` feature? It can just be zero whenever Orchard is not enabled.

Reply:
Oh, it's public. Yeah I guess all the other public APIs gate the Orchard bits on the `orchard` feature.